### ISSUE TITLE
Menu FaqCategoryMenu cannot be loaded.

### ISSUE THREAD

**Issue by divvet** [aldryn/aldryn-faq#18]
Howdy,

Just installed this with Django-CMS 3.0 however when I navigate around the CMS I continuously have this message pop up:

"Menu FaqCategoryMenu cannot be loaded. Please, make sure all its urls exist and can be resolved."

Not sure if its an error or a fault on my part but I thought I would submit it.

**Comment by mkoistinen** [comment]
@divvet Can you try again with the latest version (just posted). Note that we now use django-parler, not django-hvad.

**Comment by divvet** [comment]
I have just installed using your master branch, however I still receive that error, albeit less than I did previously. Previously every page I loaded resulted in that error, now it seems random. 

I have a few other issues since installing, and I have to admit I am not sure if I have missed something. Essentially certain displays such as "Latest Questions" results in a "No reverse match" (I can post the trace back) and others such as the "Question List" just displays a blank page despite questions being selected.
